emigration.lua

```diff
@@ -1,22 +1,29 @@
 --Allow stressed dwarves to emigrate from the fortress
 -- For 34.11 by IndigoFenix; update and cleanup by PeridexisErrant
 -- old version:  http://dffd.bay12games.com/file.php?id=8404
---@module = true
---@enable = true
```

Comment:
these two lines are required for the enabled API to function. see https://docs.dfhack.org/en/latest/docs/dev/Lua%20API.html#script-enable-api for details.